SU2_CFD/src/variables/CNEMOEulerVariable.cpp

```diff
@@ -325,9 +304,11 @@ bool CNEMOEulerVariable::Cons2PrimVar(su2double *U, su2double *V,
   }
 
   /*--- Partial derivatives of pressure and temperature ---*/
-  fluidmodel->ComputedPdU  (V, eves, val_dPdU  );
-  fluidmodel->ComputedTdU  (V, val_dTdU );
-  fluidmodel->ComputedTvedU(V, eves, val_dTvedU);
+  if(implicit){
+    fluidmodel->ComputedPdU  (V, eves, val_dPdU  );
```

Comment:
These may be required for some explicit schemes/viscous

Reply:
The rest of the code code seems okay, going to take a further look to see which schemes and routines use those quantities

Reply:
If in doubt and the computations are not expensive...